From 38f42c38838ce68f8d0a9cbdfedb2f7b7ff2ad4e Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Tue, 8 May 2018 21:10:59 -0500
Subject: [PATCH 3744/5242] MLK-18240: arm64: dts: change the i.MX8QXPB0 NAND
 iomux settings

commit  cf5c13cd512976b5194a16c4778c6869433de1ce from
https://source.codeaurora.org/external/imx/linux-imx.git

Set the corrrect NAND IOMXU for i.MX8QXPB0.

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts          |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts
index 3169b61..d116564 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts
@@ -42,8 +42,9 @@
 				SC_P_USDHC1_CMD_CONN_NAND_CE0_B		0x0e00004c
 
 				/* i.MX8QXP NAND use nand_re_dqs_pins */
-				SC_P_EMMC0_CMD_CONN_NAND_DQS		0x0e00004c
-				SC_P_USDHC1_DATA1_CONN_NAND_RE_B	0x0e00004c
+				SC_P_USDHC1_CD_B_CONN_NAND_DQS      0x0e00004c
+				SC_P_USDHC1_VSELECT_CONN_NAND_RE_B  0x0e00004c
+
 			>;
 		};
 	};
-- 
1.7.9.5

